.github/issue_template.md

```diff
@@ -23,6 +23,6 @@ PDK Version (if applicable):
 
 ### Output Log
 
-<!-- 
-For information how to capture verbose logs, look [here](https://github.com/lingua-pupuli/puppet-vscode/blob/master/client/README.md#2-capture-verbose-logs-and-send-them-to-us)
+<!--
+For information how to capture verbose logs, look [here](https://github.com/puppetlabs/puppet-vscode/blob/master/README.md#2-capture-verbose-logs-and-send-them-to-us)
```

Comment:
This should probably use our proper website?

https://puppet-vscode.github.io/docs/experience-a-problem/

Reply:
Updated